### PR TITLE
Fix for `batch_txn_tool`

### PR DESCRIPTION
**Pull request by shimkiv**
- Simple fix for `batch_txn_tool` as the "getting familiar with the `OCaml`" steps.
  - This PR only catches up the codebase to make the tool work again, there are still issues left.

### CODE REVIEW

File: src/app/batch_txn_tool/Dockerfile_txn_burst [hunk 1]
```diff
@@ -30,7 +30,6 @@ RUN eval $(opam config env) \
     src/app/archive/archive.exe \
     src/app/rosetta/rosetta.exe \
     src/app/runtime_genesis_ledger/runtime_genesis_ledger.exe \
-    src/app/generate_keypair/generate_keypair.exe \
```
Comment (mrmr1993) [inline src/app/batch_txn_tool/Dockerfile_txn_burst:33]: Why was this deleted?

File: src/app/batch_txn_tool/batch_txn_tool.ml [hunk 2]
```diff
@@ -4,9 +4,9 @@ open Signature_lib
 open Txn_tool_graphql
 open Unsigned
 
-let gen_secret_keys count =
-  Quickcheck.random_value ~seed:`Nondeterministic
-    (Quickcheck.Generator.list_with_length count Private_key.gen)
+(* let gen_secret_keys count =
```
Comment (mrmr1993) [inline src/app/batch_txn_tool/batch_txn_tool.ml:7]: If this code is unused, can we delete it?

File: src/app/batch_txn_tool/Dockerfile_txn_burst (hunk 1 above)
Comment (shimkiv) [reply src/app/batch_txn_tool/Dockerfile_txn_burst:33]: Because it duplicates line 28

**Review by mrmr1993 — APPROVED** [review]
